.github/workflows/riot-ryot.yml

```
@@ -0,0 +1,108 @@
+# Run 'compile_and_test_for_board.py' on connected boards.
+#
+# This workflow will run on a RYOT machine and lunch all tests on all boards
+# currently connected to that machine.
+#
+# Documentation:
+#
+# * Setup a RYOT machine:
+#   https://github.com/fjmolinas/riot-ryot/blob/master/setup.md
+#
+# * Requirements (filled by a RYOT machine):
+#   * ssh access (required secrets set)
+#   * all required flashing tools installed
+#   * udev rules that map all BOARD to /dev/riot/tty-$(BOARD)
+#   * RIOT_MAKEFILES_GLOBAL_PRE that sets PORT and DEBUG_ADAPTER_ID for each
+#     BOARD http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#   * Static list of connected boards or make target listing them in json format
+#     so that fromJSON can be used.
+
+name: riot-ryot
+
+on:
+  # Schedule weekly runs Saturday at 00:00 on master
+  schedule:
+    - cron: '00 0 * * 6'
+  push:
+    # TODO: remove only here to demonstrate build once
+    branches:
+      - master
+    # Run on all new release candidates
+    tags:
+      - '*-RC*'
```

Also would prefer a more stricter approach here:

```suggestion
    tags:
      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-RC[0-9]*'
      - '[0-9][0-9][0-9][0-9].[0-9][0-9]'
      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*'
```